packages/grafana-ui/src/utils/storybook/ExampleFrame.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import { Unstyled } from '@storybook/blocks';
+import { ReactNode } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+interface ExampleFrameProps {
+  children: ReactNode;
+}
+
+/**
+ * Wraps children with a border for nicer presentation in Storybook docs.
+ *
+ * This is intended to be used temporarily to move away from our <Preview> patch until
+ * we have something more long term.
```

Comment:
Longer term, is this a general strategy around how we do docs, including Examples or not?

Reply:
I'm hoping the longer term strategy would be to make stories for everything we want examples in docs for.